docs/website/contents/for-developers/HardForkCombinatorRollbackMonotonicityProposal.md

```diff
@@ -0,0 +1,285 @@
+# DRAFT NUMBER 2
+
+# Introduction
+
+The Babbage->Conway transition on an earlier iteration of SanchoNet revealed a bug in the Consensus layer by which the Hard Fork Combinator transitioned to Conway because the relevant Babbage protocol parameter update proposal received sufficient votes but then that same proposal wasn't actually enacted: the major protocol version remained unchanged even after Conway started.
+
+Troubleshooting this bug and moreover determining how to fix it in such a way that makes similar mistakes less likely in the future has lead to the following mini-framework for understanding the HFC's behavior on a particular chain.
+
+So many dimensions arose from investiating such a relatively simple misbehavior for two reasons.
+First, the mechanisms implementing these policies are---at least in the current implementation---somewhat interdependent.
+Second, the HFC (including its instantiation for Cardano) remains underdocumented since back when it was originaly developed under time-pressure, and our team no longer includes its inventor.
+The Consensus Team learns new things and unfortunately re-learns some old things every time they dive in to alter it.
+(Optimistically, Nicolas Frisby feels they currently understand it now better than they have ever before.)
+
+## Dimensions
+
+The following discrete variables characterize dimensions of the HFC design space.
+In this context, the _determination_ is the `singleEraTransition` method: it's how an era's ledger state determines when the era ends.
```

Comment:
I'm a bit confused by the reference to `singleEraTransition`? I would have expected something like: "the determination is the (single?) era transition method"

Reply:
I don't think I understand what you mean by this GitHub comment.

Reply:
Let me try again :) Why do we refer to the `singleEraTransition` identifier? Would my sentence above still be correct if we removed this?

Reply:
I think "the (single?) era transition method" is too ambiguous.

If you want to remove the identifier, then I would say:

> In this context, the _determination_ is the logic that maps a ledger state in some era to `Just e` if possible, where `e` is the `EpochNo` of the first epoch after that era.

Reply:
> I think "the (single?) era transition method" is too ambiguous.

What are the possible interpretations? (that makes this ambiguous)

I like the alternative phrasing you proposed  :+1: